### ISSUE TITLE
Can't join specific channel if name is the same

### ISSUE THREAD

**Issue by AignerM** [spdermn02/TouchPortal_Discord_Plugin#75]
Hi, i have the problem, that the plugin is using the first/random channel id for which the name matches.
We have 4 Voice channels named the same, so i can't select which one i want to join.
Maybe add a feature to join by ID and not Name (I think you already use the id just search for it by name?)

**Comment by spdermn02** [comment]
Yes that is correct I use name. I didn't really think people used the same name in their server but sure. I just added something to join dm channel by id don't see why I couldn't just change that. Or have an alternate action for join channel by server and id. I'll look into it.

**Comment by spdermn02** [comment]
yeah - that is really the only thing we can do without TP changing the way Choice lists work.
